app/controllers/support_interface/settings_controller.rb

```diff
@@ -2,13 +2,25 @@ module SupportInterface
   class SettingsController < SupportInterfaceController
     def activate_feature_flag
       FeatureFlag.activate(params[:feature_name])
-      flash[:success] = "Feature ‘#{params[:feature_name].humanize}’ activated"
+
+      SlackNotificationWorker.perform_async(
+        ":flags: Feature ‘#{feature_name}‘ was activated in production",
```

Comment:
Do we need `in production` if this can be sent from any env?

Reply:
If we want to mention production, maybe we should include the `HostingEnvironment.environment_name` instead?

Reply:
Meant to remove that! The worked already includes that for env other than production eg `[QA] Message`